modules/get-started/pages/licensing/overview.adoc

```diff
@@ -87,6 +87,10 @@ The following table lists the Enterprise features for Redpanda and how Redpanda
 | Enables compliance with FIPS security standards for cryptography.
 | No change.
 
+| xref:manage:iceberg/topic-iceberg-integration.adoc[Iceberg Topics]
+| Enables Iceberg integration for Redpanda topics.
+| Iceberg topics are disabled across the cluster. Topics cannot be created or modified with the `redpanda.iceberg.mode` property.
+
```

Comment:
Just a comment about this section. The title is **Redpanda enterprise features** (lowercase enterprise). And the the first sentence says: "The following table lists the Enterprise features." (uppercase Enterprise)